### PR TITLE
Create new account permission added

### PR DESCRIPTION
**Pull request by iArchitSharma**
**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: content/en/cloud/identity/organizations/org-permissions.md [hunk 1]
```diff
@@ -8,8 +8,9 @@ categories: [Identity]
 tags: [orgs]
 ---
 
-| Permissions                                                     | Team Admin  | Team Manager |
-|-----------------------------------------------------------------|-------------|--------------|
+| Permissions                                                     | Org Admin  | Org Billing Manager |
+|-----------------------------------------------------------------|------------|---------------|
+| Create new accounts                                             | ✅*        | ❌           |
```
Comment (leecalcote) [inline content/en/cloud/identity/organizations/org-permissions.md:13]:
```suggestion
| Create new user accounts                                             | ✅*        | ❌           |
```

@theBeginner86 is *Owner required?

Comment (iArchitSharma) [reply content/en/cloud/identity/organizations/org-permissions.md]: @leecalcote Done!

**Comment by leecalcote** [comment]
@iArchitSharma confirm regarding the Owner

**Review by theBeginner86 — APPROVED** [review]

File: content/en/cloud/identity/organizations/org-permissions.md (hunk 1 above)
Comment (theBeginner86) [reply content/en/cloud/identity/organizations/org-permissions.md]: No 👍

**Pull request merged**